### PR TITLE
Adding project phase to public

### PR DESCRIPTION
**Pull request by dhlevi**
Project phase filter added to document search.

See ticket [EE-804](https://bcmines.atlassian.net/browse/EE-804)

### CODE REVIEW

File: src/app/project/documents/documents-tab.component.ts [hunk 1]
```diff
@@ -166,10 +170,23 @@ export class DocumentsTabComponent implements OnInit, OnDestroy {
               });
             }
 
+            if (this.projectPhases.length === 0) {
+              res.documentsTableRow[0].searchResults.map(item => {
+                switch (item.type) {
```
Comment (MaxWardle) [inline src/app/project/documents/documents-tab.component.ts]: can we get rid of these switches please?

Comment (MaxWardle) [inline src/app/project/documents/documents-tab.component.ts]: can we get rid of these switches please?

**Comment by MaxWardle** [comment]
Looks good other than the switch statement thing. I tested locally and it seems to be working.

File: src/app/project/documents/documents-tab.component.ts (hunk 1 above)
Comment (dhlevi) [reply src/app/project/documents/documents-tab.component.ts]: Switches removed, and merged the statements together into a single map.

**Comment by MaxWardle** [comment]
Changes look awesome, thanks Dylan. Tested again and still looks good to me.

**Pull request merged**